front/src/applications/operationalStudies/components/SimulationResults/ChartHelpers/ChartHelpers.ts

```diff
@@ -41,8 +43,8 @@ export function offsetSeconds(seconds: number) {
   return seconds;
 }
 
-export function getDirection(data: Position[][]) {
-  const lastOfLast = last(last(data)) as Position;
+export function getDirection<T extends Position | ConsolidatedPosition>(data: T[][]) {
```

Comment:
```suggestion
export function getDirection<T extends Position>(data: T[][]) {
```

ConsolidatedPosition is an alias, so they have the same fields